### ISSUE TITLE
Visitor name not getting detected when it is added via RN SDK API

### ISSUE THREAD

**Issue by Ramanpreet6262** [zoho/SalesIQ-Mobilisten-ReactNative#45]
I am using RN SDK's API to set visitor name, email, phone number, etc.

Using like this:
`ZohoSalesIQ.setVisitorName('Test user');`

But when I initialize a chat, an operator bot asks for name and email, name is not getting auto-filled in the input box whereas for email it works fine.

And on the operator dashboard right side panel, name is shown as `Visitor 89xxx` means random allocated name and not the one I added via API, and the rest all user info is visible there as I entered.

**Version**
`  "react-native-zohosalesiq-mobilisten": "4.2.10"`

**Comment by DavidBhahirathan** [comment]
Hi @Ramanpreet6262 ,

Thank you for writing us. Please verify whether you've used the *setVisitorName* method after the SDK is initialized. I have attached the below snippet for reference. 

```
import { Platform } from 'react-native';
import { ZohoSalesIQ } from 'react-native-zohosalesiq-mobilisten';

var appKey;
var accessKey;

if (Platform.OS === 'ios') {
    appKey = "ios_appKey";
    accessKey = "ios_accessKey";
} else {
    appKey = "android_appKey";
    accessKey = "android_accessKey";
}

ZohoSalesIQ.initWithCallback(appKey, accessKey, (success) => {
    if (success) {
               ZohoSalesIQ.setLauncherVisibility(true);
               ZohoSalesIQ.setVisitorName('Name');      
    } else {
        // your code to handle initialization failure
    }
});```

And make sure the visitor name param is not null/ empty.

I hope this helps. If the issue persists, please feel free to mail us at (support@zohosalesiq.com) along with the Github issue ID.

Regards,
Bhahirathan M

**Comment by Ramanpreet6262** [comment]
Hi @DavidBhahirathan,
Yes I am using `setVisitorName` method after SDK initialisation only in the callback on success as your code ^. And visitor name param also has a valid value.

Yes following up on email. Thanks.

**Comment by DavidBhahirathan** [comment]
Hi @Ramanpreet6262,

Thank you for your patience. This issue has been addressed in version [v4.2.13](https://github.com/zoho/SalesIQ-Mobilisten-ReactNative/releases/tag/v4.2.13). Also, please check out our [latest version](https://github.com/zoho/SalesIQ-Mobilisten-ReactNative/releases/latest).

For further queries, please feel free to write us at [support@zohosalesiq.com](mailto:support@zohosalesiq.com). We will be happy to assist you.

Regards,
Bhahirathan M.

**Comment by DavidBhahirathan** [comment]
Hi @Ramanpreet6262 ,

As there is no recent activity from your side, we're closing this issue. You can reopen this again anytime. Meanwhile, please check out [latest version](https://github.com/zoho/SalesIQ-Mobilisten-ReactNative/releases/latest).

For further queries, please feel free to write us at [support@zohosalesiq.com](mailto:support@zohosalesiq.com). We will be happy to assist you.

Best regards,
Bhahirathan M.
